src/appendices/math-method-examples/random-examples.rst

```diff
@@ -0,0 +1,100 @@
+.. _random-examples:
+
+``Math.random`` Examples
+=========================
+
+The general syntax for this method is:
+
+::
```

Comment:
I think the double colon is reserved for console output.

Reply:
I have no preference for :: vs. .. sourcecode:: js, but we should check with the team before implementing a global change, since all of the method appendix pages use the :: for displaying the general syntax.